can_messages.asciipb

```diff
@@ -293,17 +293,32 @@ msg {
 
 msg {
   id: 28
-  source: DRIVER_CONTROLS
-  # target: TELEMETRY
-  msg_name: "steering angle"
+  source: DRIVER_CONTROLS_STEERING
+  # target: DRIVER_CONTROLS_CENTER_CONSOLE
+  msg_name: "steering_output"
   can_data {
     u16 {
-      field_name_1: "steering_angle"
+      field_name_1: "control_stalk_analog_state"
```

Comment:
I'm curious how are you planning on using this signal?

Is the idea here that you'll encode each of the possible tristate values for the 4 analog readings as a digital state, so you'll have `4 * 3` different values, but at most `4` set bits (and then push logic into your master to raise the appropriate local event)?

If these are intended to be interpreted as a bitset, please indicate as such by adding a bitset suffix or something for clarity.

Or are you just assuming that only 1 of the 4 analog inputs will be "active" at a time, and that the 4 are mutually exclusive?

Reply:
The idea is that control_stalk_analog_state will send out a msg containing  one of the these values (defined in exported enums) and the master board will constantly listen for this message using an rx handler and raise the appropriate event. 

{
      EE_CONTROL_STALK_ANALOG_DISTANCE_NEUTRAL,
      EE_CONTROL_STALK_ANALOG_DISTANCE_MINUS,
      EE_CONTROL_STALK_ANALOG_DISTANCE_PLUS,
  },
  {
      EE_CONTROL_STALK_ANALOG_CC_SPEED_NEUTRAL,
      EE_CONTROL_STALK_ANALOG_CC_SPEED_MINUS,
      EE_CONTROL_STALK_ANALOG_CC_SPEED_PLUS,
  },
  {
      EE_CONTROL_STALK_ANALOG_CC_DIGITAL,
      EE_CONTROL_STALK_ANALOG_CC_CANCEL,
      EE_CONTROL_STALK_ANALOG_CC_RESUME,
  },
  {
      EE_CONTROL_STALK_ANALOG_CC_TURN_SIGNAL_NONE,
      EE_CONTROL_STALK_ANALOG_CC_TURN_SIGNAL_RIGHT,
      EE_CONTROL_STALK_ANALOG_CC_TURN_SIGNAL_LEFT,
  },

Reply:
Is your message being sent on a periodic basis or immediately as physical events occur? From what I understand based on your description, this message is periodic (ie. the message gets sent after getting called in a soft timer, rather than when the physical event actually occurs).

However, if this isn't a bitset, then aren't you also implying that you can't have multiple active inputs, and only 1 analog input and 1 digital input can be active at a time? Consider if 2 physical events occur within your timer period, what would happen? If it's periodic, without using a bitset, then wouldn't you lose 1 of them?